make-options.json

```diff
@@ -3,6 +3,7 @@
         "ANT",
         "Gulp",
         "NuGetInstaller",
+        "PublishSymbols",
```

Comment:
ok the commit i'm on is actually busted already - i got confused for a while because i added console.error and so i've been spinning my wheels thinking i introduced the break :)